services/web/server/tests/unit/with_postgres/conftest.py

```diff
@@ -70,16 +73,31 @@ def app_cfg(here, osparc_simcore_root_dir):
 
     # validates and fills all defaults/optional entries that normal load would not do
     cfg_dict = trafaret_config.read_and_validate(cfg_path, app_schema, vars=variables)
+
+    # WARNING: changes to this fixture during testing propagates to other tests. Use cfg = deepcopy(cfg_dict)
```

Comment:
this is defined by the scope already.

Reply:
yeah, what I mean is that this fixture returns a dict when the session starts. A dict is a mutable object and the tests functions can change it!!! Therefore the fixture gets corrupted from one test to the next if you are not careful. My solution here was to create a new fixture with at function context that would deep copy the former and pass a fresh copy to every test